### PR TITLE
[Iteration 3] v4.0-beta

### PR DESCRIPTION
**Pull request by Jeckerson**
Issue - #102

Screen Shot 2019-08-25 at 19 41 09

Screen Shot 2019-08-25 at 19 44 55

Screen Shot 2019-08-25 at 19 44 38

Screen Shot 2019-08-25 at 19 44 11

Screen Shot 2019-08-25 at 19 44 01

Screen Shot 2019-08-25 at 19 43 47

Screen Shot 2019-08-25 at 19 43 24

Screen Shot 2019-08-25 at 19 43 11

Screen Shot 2019-08-25 at 19 42 54

Screen Shot 2019-08-25 at 19 42 34

Screen Shot 2019-08-25 at 19 42 02

### CODE REVIEW

File: app/controllers/ProfilesController.php [hunk 1]
```diff
@@ -88,7 +88,7 @@ public function createAction()
             ]);
 
             if (!$profile->save()) {
-                $this->flash->error($profile->getMessages());
+                $this->flash->error(join('<br/>', $profile->getMessages()));
```
Comment (ruudboon) [inline app/controllers/ProfilesController.php]: Why not use [nl2br](https://www.php.net/manual/en/function.nl2br.php)

File: app/views/about/index.volt [hunk 2]
```diff
@@ -1,14 +1,8 @@
...
-		This is a sample application for the <a href="http://phalconphp.com">Phalcon Framework</a>. We expect to implement as many features
-		as possible to show how the framework works and its potential. Please write us if you have any
-		feedback or comments. Feel free to clone the code of this application <a href="https://github.com/phalcon/vokuro">here</a>.
-		The purpose of this application is merely academic. Thanks!.
-	</div>
-
-</div>
\ No newline at end of file
+<div>
+    This is a sample application for the <a href="http://phalconphp.com">Phalcon Framework</a>. We expect to implement as many features
```
Comment (ruudboon) [inline app/views/about/index.volt]: I think we can start using phalcon.io as our main domain. CC @niden

File: app/views/emailTemplates/confirmation.volt [hunk 3]
```diff
@@ -1,24 +1,25 @@
-
-
 <tbody>
-	<tr>
-		<td style="padding:40px 0  0 0;">
-			<p style="color:#000;font-size: 16px;line-height:24px;font-family:'HelveticaNeue','Helvetica Neue',Helvetica,Arial,sans-serif;font-weight:normal;">
+    <tr>
+        <td style="padding:40px 0  0 0;">
```
Comment (ruudboon) [inline app/views/emailTemplates/confirmation.volt]: Is inline styling still needed for email, or can we move this to the head?

File: app/views/layouts/emailTemplates.volt [hunk 4]
```diff
@@ -1,50 +1,49 @@
...
+    <!--footer-->
+    <tbody>
+    <tr>
+        <td align="right" style="padding:25px 0  0 0;">
+            <table border="0" cellspacing="0" cellpadding="0" style="padding-bottom:9px;" align="right">
+                <tbody>
+                <tr style="border-bottom:1px solid #999999;">
+                    <td width="24" style="padding:0 7px 0 0;">
+                        <a href="http://www.facebook.com/pages/Phalcon/134230726685897" style="border-width:0;">
+                            <img src="http://{{ publicUrl }}/img/facebook.gif" width="24" height="24" alt="Facebook Image">
```
Comment (ruudboon) [inline app/views/layouts/emailTemplates.volt]: Can we add http(s) detection? If you are running on https this won't work.

File: app/views/users/search.volt [hunk 5]
```diff
@@ -33,26 +27,33 @@
             <td>{{ user.banned == 'Y' ? 'Yes' : 'No' }}</td>
             <td>{{ user.suspended == 'Y' ? 'Yes' : 'No' }}</td>
             <td>{{ user.active == 'Y' ? 'Yes' : 'No' }}</td>
-            <td width="12%">{{ link_to("users/edit/" ~ user.id, '<i class="icon-pencil"></i> Edit', "class": "btn") }}</td>
-            <td width="12%">{{ link_to("users/delete/" ~ user.id, '<i class="icon-remove"></i> Delete', "class": "btn") }}</td>
+            <td style="width: 12%;">{{ link_to("users/edit/" ~ user.id, '<i class="icon-pencil"></i> Edit', "class": "btn btn-sm btn-outline-warning") }}</td>
```
Comment (ruudboon) [inline app/views/users/search.volt]: Can we keep the styling in the css file?

File: app/views/emailTemplates/confirmation.volt (hunk 3 above)
Comment (Jeckerson) [reply app/views/emailTemplates/confirmation.volt]: Yes, for backward compatibility.

File: app/views/layouts/emailTemplates.volt (hunk 4 above)
Comment (niden) [reply app/views/layouts/emailTemplates.volt]: Yes and let us use the phalcon.link URL shorteners

File: app/controllers/ProfilesController.php (hunk 1 above)
Comment (Jeckerson) [reply app/controllers/ProfilesController.php]: 968b771

File: app/views/layouts/emailTemplates.volt (hunk 4 above)
Comment (Jeckerson) [reply app/views/layouts/emailTemplates.volt]: 3fdd167

File: app/views/users/search.volt (hunk 5 above)
Comment (Jeckerson) [reply app/views/users/search.volt]: 86d4fcf

File: app/views/about/index.volt (hunk 2 above)
Comment (niden) [reply app/views/about/index.volt]: Yes please replace phalconphp.com with phalcon.io.

Comment (Jeckerson) [reply app/views/about/index.volt]: 77f064b

File: app/views/layouts/emailTemplates.volt (hunk 4 above)
Comment (Jeckerson) [reply app/views/layouts/emailTemplates.volt]: d8b5d2f

**Pull request merged**